main/guides/getting-started/README.md

```diff
@@ -293,8 +293,15 @@ Once your wallet is connected, click on the "Make Offer" button to purchase 3 pr
 
 ![Making an offer](./assets/new_004.png)
 
-When the transaction is complete you will notice some IST has been debited from your wallet, and you are the owner of three new properties.
+When the transaction is complete you will notice some IST has been debited from your wallet, and you are the owner of three new properties. Feel free to experiment with offering less than the required 0.25 IST or attemping to purchase more than three items. You'll see these offers will not be accepted. Conditional offers like this are a key part of the Agoric platform!
 
 ![Finished transaction](./assets/new_006.png)
 
-Congratulations! You got your first Agoric dapp up and running!
+Congratulations! You got your first Agoric dapp up and running! Now that you've got your first Agoric dapp up and running, let's focus on some key takeaways from this exercise:
+- Starting an Agoric Chain: You have learned the process to initialize and start a local Agoric blockchain. 
+
+- Deploying a Contract: You have learned the steps to deploy a smart contract onto the Agoric platform!
+
+- Making an Offer: Finally, you've learned to make an offer and seen the conditional nature of the offers that can be made on the Agoric platform. As you saw, when attempting to submit less than 0.25 IST or purchase more than three items the offer is rejected.
```

Comment:
I like the "Feel free to experiment ..." bit above, but the key take-away is not that other offers are rejected. The contract itself could be doing that. The key takeaway is that **the constraints they sign are enforced by the platform**.

The ui can't spoof them by showing a 25 cent charge but really charging 500 IST - that would show up in the Keplr dialog. Nor can the contract take the 0.25 cents without satisfying the want constraint. And neither can the contract take more than the 0.25 cents in the give contraint.

Reply:
Leaving out the "As you saw..." sentence would address my concern for this PR.

Reply:
Points about the "Making an Offer" bullet in Summary section addressed